.github/workflows/action.yml

```diff
@@ -22,11 +22,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Setup dotnet 6.0.x
-        uses: actions/setup-dotnet@v2
+        uses: actions/setup-dotnet@v3
         with:
           dotnet-version: "6.0.x"
       - name: Setup dotnet 7.0.x
-        uses: actions/setup-dotnet@v2
```

Comment:
Hi @sensslen,

A month ago, I tried to update the release.yml to the latest one, but the release was failing. I will merge it but keep in mind that you might need to use a specific version of .NET 7.0.103 to work.

Thanks